source/gloperate/source/base/GLContextUtils.cpp

```diff
@@ -16,6 +16,15 @@ namespace gloperate
 {
 
 
+namespace
+{
+// these are not part of the OpenGL specification, but defined identically
+// in all platform-specific context creation APIs
+const auto GL_CONTEXT_CORE_PROFILE_BIT          = 0x1;
```

Comment:
Actually, these flags are defined by OpenGL 3.2 (see gl.xml) and above and is available in every glbinding/*/bitfield.h (https://github.com/cginternals/glbinding/blob/master/source/glbinding/include/glbinding/gl32core/bitfield.h#L14). I'm still not sure why a google search for the flags doesn't result in anything meaningful.